pkg/debugapi/info.go

```diff
@@ -0,0 +1,40 @@
+package debugapi
+
+import (
+	"net/http"
+
+	"github.com/ethersphere/bee/pkg/jsonhttp"
+)
+
+type BeeNodeMode uint
+
+const (
+	LightMode BeeNodeMode = iota
+	FullMode
+	DevMode
+)
+
+type infoResponse struct {
+	BeeMode     string `json:"beeMode"`
+	GatewayMode bool   `json:"gatewayMode"`
+}
+
+func (b BeeNodeMode) String() string {
+	switch b {
+	case LightMode:
+		return "light"
+	case FullMode:
+		return "full"
+	case DevMode:
+		return "dev"
+	}
+	return "unknown"
+}
+
+// infoGetHandler gives back information about the Bee node configuration.
+func (s *Service) infoGetHandler(w http.ResponseWriter, r *http.Request) {
+	jsonhttp.OK(w, infoResponse{
+		BeeMode:     s.beeMode.String(),
```

Comment:
why not use the existing api endpoint that shows the versions?

Reply:
this is still not addressed, also, `info` is very general name for such an endpoint. not sure why a new endpoint is needed

Reply:
well, the JS team hadn't found corresponding existing endpoint for these properties, moreover the `apiversion`, `debugApiVersion` and the `version` properties also could be placed on this new EP. Thereby, the `health` EP could act as a healthcheck only and all other informational parameters could be served from the `info` EP.